src/iarc7_motion/iarc_tasks/velocity_task.py

```diff
@@ -51,7 +51,12 @@ def __init__(self, task_request):
         self._HORIZ_X_VEL = task_request.x_velocity
         self._HORIZ_Y_VEL = task_request.y_velocity
 
-        self._z_holder = HeightHolder()
+        if task_request.z_position > 0.00001:
```

Comment:
I'm sure there is a succinct Pythonic-way of doing this, no @amiller27 ?

Reply:
Succinct way of doing what? This looks fine to me

Reply:
Checking for zero

Reply:
He's not checking for zero though

Reply:
But why not check for zero?

Reply:
Fixed